### PR TITLE
Improve description of cas input/output formats.

### PR DESCRIPTION
**Pull request by sebras**
Correct details about input/output formats and provide more
thorough description of character example.

### CODE REVIEW

File: misc/cas_syntax.txt [hunk 1]
```diff
@@ -14,10 +14,11 @@ v means following moves are visible.
 r means end, output label=x-b+number.
 
 All other commands are character pairs denoting y,x coordinates in a
-12x14 character raster.  Draw a line to that coordinate.  Vertical
-coordinates are a-l, and horizontal are a-m.  aa is upper left and mn
-is lower right.  All characters end with a "x mn r" sequence to move
-to the beginning of the next character.
+14x16 character raster.  Draw a line to that coordinate.  Vertical
+coordinates are a-n, and horizontal are a-p.  aa is upper left and pn
```
Comment (larsbrinkhoff) [inline misc/cas_syntax.txt]: Swap vertical/horizontal.

**Comment by larsbrinkhoff** [comment]
Generally this looks like an improvement.  I didn't check details.

File: misc/cas_syntax.txt (hunk 1 above)
Comment (sebras) [reply misc/cas_syntax.txt]: Ah, yes. I'll fix it!

**Comment by sebras** [comment]
> Generally this looks like an improvement. I didn't check details.

Thanks for spending time reviewing. :)

File: misc/cas_syntax.txt [hunk 2]
```diff
@@ -53,6 +76,27 @@ This character is labelled "lbk" (left bracket).  Move invisibly to
 "ai" near upper right.  Draw line left to "af".  Draw down to "mf".
 Draw right to "mj".  Invisibly move to "mn" beoynd lower right.  End.
 
+This can be interpreted in a grid starting a location ma (row m, column a)
+denoted by a 0, and each succeeding move indicated by an increaing number:
```
Comment (dspinellis) [inline misc/cas_syntax.txt]: increasing

File: misc/cas_syntax.txt [hunk 3]
```diff
@@ -69,3 +113,56 @@ means draw down four steps.  72 means draw left four steps.  62 means
...
 
 0236000 means there are 15 moves, or 6-bit bytes.
+
+0236000 in binary is 010011110000000000, and shifting this value
+10 bits to the right gives 01001111, where only the 6 least significant
+bits indicate the number of moves, i.e. 001111 or 15 in decimal.
+
+The following explanation separates out each move onto its own line.
+Further it decodes each move into binary, and separates out and interpretes
+the three distance, visibility and direciton fields.  Finally eaach line
```
Comment (dspinellis) [inline misc/cas_syntax.txt]: each

Comment (sebras) [reply misc/cas_syntax.txt]: Thanks for pointing these out! This prompted me to run it through aspell causing me to find another 7 typos. :)

**Pull request merged**